modules/nf-core/seqtk/cutn/tests/main.nf.test

```diff
@@ -0,0 +1,61 @@
+nextflow_process {
+
+    name "Test Process SEQTK_CUTN"
+    script "../main.nf"
+    process "SEQTK_CUTN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "seqtk"
+    tag "seqtk/cutn"
+
+    test("homo_21_cut") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                                [ id:'test', single_end:true ],
+                                file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+                            ]
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert snapshot(process.out).match() },
+                { assert snapshot(
+                            file(process.out.bed[0][1]).name
+                                ).match("genome_cut")
+                }
```

Comment:
I don't think you need that if you already snapshot `process.out`

Reply:
I thought that the above was specifically for the file where as the `process.out` was there to catch everything. Happy to be wrong and remove the `process.out` though.

Reply:
Hi @maxulysse,
You were right, so i've updated to just use process out as that is better for static output. The other method is for when output has content varying between runs. Just a misunderstanding on my part to include both.

There's now an issue with the runner but that's under investigation.